src/User/resources/i18n/ru/usuario.php

```diff
@@ -302,4 +302,12 @@
     'Unable to disable two-factor authorization.' => '@@Не удалось отключить двухфакторную авторизацию.@@',
     'We couldn\'t re-send the mail to confirm your address. ' => '@@Мы не можем повторно отправить письмо для подтверждения вашего адреса электронной почты.@@',
     'We have sent confirmation links to both old and new email addresses. ' => '@@Мы отправили письма на ваш старый и новый почтовые ящики. Вы должны перейти по обеим, чтобы завершить процесс смены адреса.@@',
+    'User agent' => '',
```

Comment:
missing translation?

Reply:
This phrase has not correct translation on russian and people use at it is. 

Should i replace '' on 'User agent'?

Reply:
I think it's better